v2/inbound_call.go

```diff
@@ -81,21 +81,12 @@ func (ic *InboundCall) ReadFromRequest(req *Request) error {
 	return nil
 }
 
-// ReadFromRequestMeta reads information from the given request.
-//
-// This information may be queried on the context using functions like Caller,
-// Service, Procedure, etc.
-func (ic *InboundCall) ReadFromRequestMeta(reqMeta *RequestMeta) error {
-	ic.req = reqMeta.ToRequest()
-	return nil
-}
-
 // WriteToResponse writes response information from the InboundCall onto the
 // given Response.
 //
 // If used, this must be called before writing the response body to the
 // Response.
-func (ic *InboundCall) WriteToResponse(res *Request) {
+func (ic *InboundCall) WriteToResponse(res *Response) {
```

Comment:
Was this just blatantly wrong before?

Reply:
my mistake, this was changed incorrectly in the previous commit